model/ResultsService.php

```diff
@@ -53,12 +54,121 @@ class ResultsService extends tao_models_classes_ClassService
     const VARIABLES_FILTER_LAST_SUBMITTED = 'lastSubmitted';
     const VARIABLES_FILTER_FIRST_SUBMITTED = 'firstSubmitted';
 
+    const PERSISTENCE_CACHE_KEY = 'resultCache';
+
     /**
      *
      * @var \taoResultServer_models_classes_ReadableResultStorage
      */
     private $implementation = null;
 
+    /**
+     * Internal cache for item info.
+     *
+     * @var array
+     */
+    private $itemInfoCache = [];
+
+    /**
+     * External cache.
+     *
+     * @var \common_persistence_KvDriver
+     */
+    private $resultCache;
+
+    /**
+     * @return \common_persistence_KvDriver|null
+     */
+    public function getCache()
+    {
+        if (is_null($this->resultCache)) {
+            /** @var \common_persistence_Manager $persistenceManager */
+            $persistenceManager = $this->getServiceLocator()->get(\common_persistence_Manager::SERVICE_ID);
+            if ($persistenceManager->hasPersistence(self::PERSISTENCE_CACHE_KEY)) {
+                $this->resultCache = $persistenceManager->getPersistenceById(self::PERSISTENCE_CACHE_KEY);
+            } else {
+                common_Logger::i('Set a persistence with key "'. self::PERSISTENCE_CACHE_KEY .'" in order to cache result data.');
```

Comment:
I'm afraid this will be terribly noisy for our devops. I would say that messages to developers like this should not be part of logging, but rather documentation or comments in configuration files. Could you please remove this line?